CHANGELOG.rst

```diff
@@ -101,6 +101,90 @@ Added
 
 Changed
 ^^^^^^^
+* `@senwu`_: Switch to Emmental as the default learning engine.
+
+.. note::
+
+    Instead of owning Fonduer learning engine, we switch to Emmental which is a deep learning
+    framework for multi-task learning. This change can give Fonduer more power to support more
+    applications. Example usage:
```

Comment:
Suggest to reword to something like:

> Rather than maintaining a separate learning engine, we switch to Emmental, a deep learning framework for multi-task learning. Switching to a more general learning framework allows Fonduer to support more applications and multi-task learning. Example usage: